### ISSUE TITLE
C++ Support

### ISSUE THREAD

**Issue by skyestudiosDev** [Utkarsh-Deshmukh/Fingerprint-Enhancement-Python#28]
Is there a C++ version of this or you plan to write one?

**Comment by Utkarsh-Deshmukh** [comment]
There isn't a C++ version of this in this repo. there are currently no plans of developing one. Please let me know if this is really needed, and if yes, then I can plan to work on it.

**Comment by skyestudiosDev** [comment]
Yes, we wanted to use it in a cross mobile application and we figured C++ will be best

**Comment by Utkarsh-Deshmukh** [comment]
Currently there is no plan of migrating this project over to C++. So unfortunately, I cannot give you any estimate dates for this task
